util/yang_test.go

```diff
@@ -1019,6 +1019,33 @@ func TestIsYgotAnnotation(t *testing.T) {
 	}
 }
 
+func TestIsYangPresence(t *testing.T) {
+	type testStruct struct {
+		Yes *string `yangPresence:"true"`
+		No  *string
+	}
+
+	tests := []struct {
+		name string
+		in   reflect.StructField
+		want bool
+	}{{
+		name: "yangPresence container/field",
+		in:   reflect.TypeOf(testStruct{}).Field(0),
```

Comment:
Ack, Thanks Wen!

Reply:
Hi Wen,  `FieldByName(name string) (StructField, bool)` returns 2 values, so I'd need to assign a variable first...
The test cases for `IsYgotAnnotation()` work similar using `reflect.typeOf(struct{}).Field(int)`, 

Or would you rather have me define:
```		
    in: reflect.StructField{
        Name: "Yes",
        Tag:  `yangPresence:"true"`,
    },
```
and not define a testStruct{} at all?

Reply:
I went with `FieldByName` anyhow, as this is resembles the real scenario as close as possible. Thnx!

Reply:
Oh i see I was looking at reflect.Value's `FieldByName`. It's a nit so since it's not convenient feel free to keep to whatever you like. I don't mind.